pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/bootstrap-sources.nix

```diff
@@ -27,8 +26,8 @@ rec {
   # To build see `make-bootstrap-sources.nix`
   src = import <nix/fetchurl.nix> {
     inherit name;
-    url = "https://github.com/emilytrau/bootstrap-tools-nar-mirror/releases/download/2023-05-02/${name}.nar.xz";
-    hash = "sha256-ZRG0k49MxL1UTZhuMTvPoEprdSpJRNVy8QhLE6k+etg=";
+    url = "https://github.com/emilytrau/bootstrap-tools-nar-mirror/releases/download/2023-05-18/${name}.nar.xz";
```

Comment:
is replacing this with a `tarballs.nixos.org` address to be a follow up PR after `minimal-bootstrap-sources` is in all-packages?

Reply:
It's a placeholder to be replaced before this PR is merged :>